app/state/stateSvc.js

```diff
@@ -190,6 +190,7 @@ function stateSvc(
       // Initialize empty arrays for storypins
       svc.getConfig().storypins = [[]];
       // TODO: Do the same for storyframes
+      // svc.getConfig() = [[]];
```

Comment:
Can this commented out code go or is it meant to be used?

Reply:
Yes, removed.